packages/ui/src/components/App.tsx

```diff
@@ -524,6 +524,29 @@ const Alerts = observer((props: AppProps) => {
   if (props.errors.lastError == null) return null;
   // Not using the `data-dismiss="alert"` to dismiss via Bootstrap JS
   // becuase then it doesn't re-render when there's a new error.
+  console.log('props.deploy', props.deploy);
+  if (
+    props.deploy.deploy === null &&
+    props.deploy.deployHashBase16?.includes('deploy')
```

Comment:
What is the deployHashBase16 supposed to be? Why does it include `deploy`?

Reply:
This part is for the specified message when we don't get data for the deploy page. I think it's not the best approach to specify messages for each page.